ios/src/ImageCropPicker.m

```diff
@@ -908,4 +723,403 @@ - (void)cropViewController:(TOCropViewController *)cropViewController didFinishC
+                    UIImage *image = [[UIImage alloc] initWithData:imageData];
+
+                    ImageResult *imageResult = [self makeResultFromImageData:imageData image:image];
+
+                    NSDictionary *attachment = [self createAttachmentResponse:targetURL.absoluteString
+                                                                     withExif:exif
+                                                                withSourceURL:targetURL.absoluteString
+                                                          withLocalIdentifier:provider.suggestedName
+                                                                 withFilename:url.lastPathComponent
+                                                                    withWidth:imageResult.width
+                                                                   withHeight:imageResult.height
+                                                                     withMime:imageResult.mime
+                                                                     withSize:@(imageResult.data.length)
+                                                                 withDuration:nil
+                                                                     withData:[self.options[@"includeBase64"] boolValue] ? [imageResult.data base64EncodedStringWithOptions:0] : nil
+                                                                     withRect:CGRectNull
+                                                             withCreationDate:[self dateForFileAtURL:url key:NSURLCreationDateKey]
+                                                         withModificationDate:[self dateForFileAtURL:url key:NSURLContentModificationDateKey]];
+                    [selection addObject:attachment];
+                    processed++;
+                    [lock unlock];
+
+                    if (processed == results.count) {
+                        dispatch_async(dispatch_get_main_queue(), ^{
+                            [indicatorView stopAnimating];
+                            [overlayView removeFromSuperview];
+                            [picker dismissViewControllerAnimated:YES completion:[self waitAnimationEnd:^{
+                                self.resolve(selection);
+                            }]];
+                        });
+                    }
+                }];
+            }
+        }
+    }];
+}
+
+- (void)makeResponseForURL:(NSURL *)url completion:(void (^)(NSDictionary* video))completion {
+    NSURL *targetURL = [self copyFileFromURL:url];
+    AVURLAsset *asset = [AVURLAsset assetWithURL:targetURL];
+    NSString *fileName = url.lastPathComponent;
+    [self handleVideo:asset withFileName:fileName withLocalIdentifier:nil completion:completion];
+}
+
+- (NSURL *)copyFileFromURL:(NSURL *)url {
+    NSString *path = [[NSTemporaryDirectory() stringByStandardizingPath] stringByAppendingPathComponent:url.lastPathComponent];
```

Comment:
`PHPickerViewController` runs out of process. To be able to access the file it needs to be copied over. The documentation doesn't mention this but see https://developer.apple.com/forums/thread/654898 for the discussion (and my local tests showed the same, you can't do anything with the files without copying them first)